packages/components/src/images/icons/early-access.svg

```diff
@@ -0,0 +1,7 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
```

Comment:
yeah, it was kind of intentionally, this one is for navigation bar, the other (from suite-data) is for Early Access modal. But I could rename this one maybe to `experimental-features.svg` because it's meant for possible future features...